library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+    * Pediatric-episode <nowiki> {suggestedTag=Finding-significance-to-recording, suggestedTag=Episode-consciousness-affected, suggestedTag=Episode-awareness, suggestedTag=Episode-prodrome, suggestedTag=Episode-tongue-biting}[(Source: Beniczky ea 2017, Table 9.)] </nowiki>
+        ** Hyperekplexia <nowiki> [Disorder characterized by exaggerated startle response and hypertonicity that may occur during the first year of life and in severe cases during the neonatal period. Children usually present with marked irritability and recurrent startles in response to handling and sounds. Severely affected infants can have severe jerks and stiffening, sometimes with breath-holding spells. (Source: Beniczky ea 2017, Table 9; Beniczky ea 2013, Appendix S5.)] </nowiki>
+        ** Jactatio-capitis-nocturna <nowiki> [Relatively common in normal children at the time of going to bed, especially during the first year of life, the rhythmic head movements persist during sleep. Usually, these phenomena disappear before 3 years of age. (Source: Beniczky ea 2017, Table 9; Beniczky ea 2013, Appendix S5.)]</nowiki>
+        ** Pavor-nocturnus <nowiki> [A nocturnal episode characterized by age of onset of less than five years (mean age 18 months, with peak prevalence at five to seven years), appearance of signs of panic two hours after falling asleep with crying, screams, a fearful expression, inability to recognize other people including parents (for a duration of 5-15 minutes), amnesia upon awakening. Pavor nocturnus occurs in patients almost every night for months or years (but the frequency is highly variable and may be as low as once a month) and is likely to disappear spontaneously at the age of six to eight years. (Source: Beniczky ea 2017, Table 9; Beniczky ea 2013, Appendix S5.)]</nowiki>
+        ** Pediatric-stereotypical-behavior-episode <nowiki> [Repetitive motor behavior in children, typically rhythmic and persistent; usually not paroxysmal and rarely suggest epilepsy. They include headbanging, head-rolling, jactatio capitis nocturna, body rocking, buccal or lingual movements, hand flapping and related mannerisms, repetitive hand-waving (to self-induce photosensitive seizures).(Source: Beniczky ea 2017, Table 9; Beniczky ea 2013, Appendix S5.)]</nowiki>
+    * Paroxysmal-motor-event <nowiki> {suggestedTag=Finding-significance-to-recording, suggestedTag=Episode-consciousness-affected, suggestedTag=Episode-awareness, suggestedTag=Episode-prodrome, suggestedTag=Episode-tongue-biting} [Paroxysmal phenomena during neonatal or childhood periods characterized by recurrent motor or behavioral signs or symptoms that must be distinguishes from epileptic disorders. (Source: Beniczky ea 2017, Table 9; Beniczky ea 2013, Appendix S5.)]</nowiki>
```

Comment:
that must be distinguishes from  --> that must be distinguished from

Reply:
fixed